synapse/storage/databases/main/account_data.py

```diff
@@ -122,9 +122,9 @@ def get_max_account_data_stream_id(self) -> int:
         return self._account_data_id_gen.get_current_token()
 
     @cached()
-    async def get_account_data_for_user(
+    async def get_global_account_data_for_user(
         self, user_id: str
-    ) -> Tuple[Dict[str, JsonDict], Dict[str, Dict[str, JsonDict]]]:
+    ) -> Dict[str, JsonDict]:
```

Comment:
Can we be explicit that this is _global_ account data in the docstring? (+129, can't seem to add a comment to that line)